index.html

```diff
@@ -9,7 +9,10 @@
   <link rel="stylesheet"  type="text/css" href="application.css">
 </head>
 <body class="container">
-  <img class="container-item book-cover" src="resources/book-cover.png" alt="book_cover">
+  <div class="container-item">
+    <img class="book-cover" src="resources/book-cover.png" alt="book_cover">
```

Comment:
### 
```suggestion
    <img class="book-cover" src="resources/book-cover.png" alt="book_cover" />
```

Reply:
this tag don’t need this 
look :eyes: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img

Reply:
But in wolox is mandatory, Frontend Gabo words.